setup.py

```diff
@@ -44,7 +44,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.10",
+        "pytket == 1.11.0rc0",
```

Comment:
The CI should pick that up anyway, no need to update this here?

Reply:
Ah yes you are right.

Reply:
Actually it seems not in this case -- there is _no_ matching pytket 3.11 wheel for the initial install, so the build fails.

Reply:
Oh, okay.